pydax/loaders/_table.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2020 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"Tabular data loaders."
+
+import os
+
+from typing import Dict, Optional, Union
+
+import pandas as pd  # type: ignore
+
+from .. import _typing
+from ..schema import SchemaDict
+
+from ._base import Loader
```

Comment:
```suggestion
import os
from typing import Dict, Optional, Union

import pandas as pd  # type: ignore

from .. import _typing
from ..schema import SchemaDict
from ._base import Loader
```
Is there a reason for the spaces? I figure we group as 1. Standard libs, 2. External libs, 3. PyDAX modules

Reply:
Yes, that's how I grouped

Reply:
Followed [PEP8's suggestions](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:

>    Standard library imports.
>    Related third party imports.
>    Local application/library specific imports.